src/main/java/uk/gov/justice/digital/hmpps/cmd/api/model/DryRunNotification.kt

```diff
@@ -56,13 +56,13 @@ data class DryRunNotification(
 
     fun fromDto(dtoCsr: CsrModifiedDetailDto): DryRunNotification {
       return DryRunNotification(
-        quantumId = dtoCsr.quantumId,
-        shiftModified = dtoCsr.shiftModified,
+        quantumId = dtoCsr.quantumId!!,
+        shiftModified = dtoCsr.shiftModified!!,
```

Comment:
not sure I understand why it is nullable when here you assume it is not nullable?